.github/workflows/push-to-docs.yml

```
@@ -0,0 +1,55 @@
+
+jobs:
+  publish-to-docs:
+    # Roda apenas se o PR foi aceito (mergeado) e não apenas fechado/rejeitado
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Create Blog Post Content
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)